processor/elastictraceprocessor/metadata.yaml

```diff
@@ -0,0 +1,12 @@
+type: elastictrace
+scope_name: github.com/elastic/opentelemetry-collector-components/processor/elastictraceprocessor
+
+status:
+  class: processor
+  stability:
+    deprecated: [traces]
+  distributions: []
+  deprecation:
+    traces:
+      date: "2025-10-08"
```

Comment:
This is TBD - it must be a string, otherwise it's rejected. I'd wanted to have a string saying, something like "since Elastic stack release 9.2", but we can't have that, so we need a date.